src/styles/font_names.js

```diff
@@ -3,6 +3,9 @@ const fonts = {
     space_mono: "Space Mono",
     itc_avante_garde: "itc-avant-garde-gothic-pro",
     footerSizeDesktop: 20,
+    LargeBook: 27,
+    Heading2Size: 60,
+    BodySize: 18,
```

Comment:
if you're going to be adding in the fonts this way can you save them as strings (preferably in `ems` https://nekocalc.com/px-to-em-converter) so you don't have to go back and change all of this later on? then you can call them in the code without having to add the `px` at the end